src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/Details.cshtml

```diff
@@ -410,7 +410,7 @@
     <p>
         @if (Model.ShowAddAnotherApprenticeOption)
         {
-            <a class="govuk-link add-apprentice" href="@Url.Action("AddAnotherSelectMethod", "DraftApprenticeship", new { Model.CohortReference, Model.ProviderId, Model.AccountLegalEntityHashedId, Model.EncodedPledgeApplicationId, Model.TransferSenderHashedId, UseLearnerData = Model.UseLearnerData })">
+            <a class="govuk-link add-apprentice" href="@Url.Action("SelectLearnerRecord", "Learner", new { Model.ProviderId, Model.CohortReference, Model.AccountLegalEntityHashedId, UseLearnerData = true })">
                 Add another apprentice
             </a>
```

Comment:
This is going to the wrong page. It needs to arrive at GetReservationId in DraftApprenticeship. At this point it;s then sent to reservations to get a reservationId